README.md

```diff
@@ -90,6 +90,63 @@ found 1725 well-typed programs.
 </pre>
 </details>
 
+## How to validate claims from the paper?
+
+This section lists claims that we made in the paper, and where in this repository to find the corresponding evidences.
+
+> (page 10; Section 4) The model covers a substantial part of the Python language including assertions, loops, exception handlers, and delete statements.
+
+You might use `grep` to validate that we include those language constructs in [./conformance_suite](./conformance_suite). For example, to see whether we tested assertions, you can run
+
+```bash
+$ grep 'assert ' -r ./conformance_suite/
+```
+
+To further validate that those tests are indeeded tested, you can change the source code such that the test should fail, and redo `make` to see if the test does fail.
+
+> (page 10; Section 4) Second, we used Redex’s random testing tools to check type soundness on thousands of examples (1,600 expressions and 11,000 programs).
+
+We have two uses of [`redex-check`](https://docs.racket-lang.org/redex/reference.html#%28form._%28%28lib._redex%2Freduction-semantics..rkt%29._redex-check%29%29) in [./conjectures.rkt](./conjectures.rkt) (line [172](https://github.com/brownplt/insta-model/blob/c9f21f5479b2dd4f9ddbfabacba88d22b3cb1811/conjectures.rkt#L172) and [239](https://github.com/brownplt/insta-model/blob/c9f21f5479b2dd4f9ddbfabacba88d22b3cb1811/conjectures.rkt#L239)).
+
+We will walk you through the first use. The other is very similar. On line 172 you will see the following expression. The expression generate 10000 random expressions (`e+`) and checks whether for all expression `e+`, if the expression compiles and terminates within certain amoount of reduction steps, the result must be of the same type as the expression. The nonterminate `e+` is defined in language `SP-conjecture`, which inherite the definition from language `SP` ([line 29 of `grammar.rkt`](https://github.com/brownplt/insta-model/blob/c9f21f5479b2dd4f9ddbfabacba88d22b3cb1811/grammar.rkt#L29)
+
+```racket
+(redex-check SP-conjecture
+             e+
+             (term (compile-implies-terminate-implies-well-typed-e e+))
+             #:attempts 10000)
+```
+
+In the repository we tested 10000 random expressions and 15000 random programs. The numbers are different in the paper (1600 and 11000) but you can always change the numbers.
```

Comment:
The numbers in the paper are about **well-typed** terms, not random terms.
(I'll reword this).